extensions/ql-vscode/src/local-queries/qlpack-generator.ts

```diff
@@ -76,7 +76,10 @@ export class QlPackGenerator {
   }
 
   private async createWorkspaceFolder() {
-    await mkdir(this.folderUri.fsPath);
+    const folderExists = await pathExists(this.folderUri.fsPath);
+    if (!folderExists) {
+      await mkdir(this.folderUri.fsPath);
+    }
```

Comment:
This should be equivalent (although you'll need to add the import):

```suggestion
    await ensureDir(this.folderUri.fsPath);
```

Reply:
Ah nice, thanks for the tip!